.gitpod.yml

```diff
@@ -0,0 +1,20 @@
+tasks:
+  - before: |
+      sudo apt update -y
+      DEBIAN_FRONTEND=noninteractive sudo apt-get install -y bison python3-dev libcppunit-dev build-essential cmake autoconf tcl pkg-config git curl patch libtool-bin
```

Comment:
make sure python3 is the default python interpreter

Reply:
sure, the default image [gitpod/workspace-full](https://hub.docker.com/r/gitpod/workspace-full) sets python3 as default